app/views/lesson_plans/show.pdf.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Lesson Plan:</h1>
+
+<% @lesson_plan.lessons.each_with_index do |lesson, ilesson| %>
+  <%= ilesson + 1 %>. <%= lesson.name %> (<%= lesson.duration.in_words %>)<br />
```

Comment:
nit: is `i#{object}` a convention we use?  If not, maybe just `i`.
**This is a tiny tiny nit pls feel free to ignore it**